docker-compose.yml

```diff
@@ -1,9 +1,9 @@
 version: '3.2'
 services:
-#  db:
-#    image: postgres
-#    volumes:
-#      - ./tmp/db:/var/lib/postgresql/data
+  db:
+    image: postgres
```

Comment:
Can you have a look and see if there's a way to specify the postgres version? Would be good to lock that in to the same as the PaaS version ie 10.

Reply:
postgres:10 is the name of the image and the version separated by a colon which ive now implemented

Reply:
So turns out we want postgres:11, but we can set it to psotgres:latest?

Reply:
Oh, I'm confused: we chose `tiny-unencrypted-10` for the PaaS service, so I think 10 is fine?

Reply:
When trying to run it, it says it needs the latest version of postgres which is 11

Reply:
In which case I wonder how this works when we deploy to PaaS? We need these to be consistent, somehow, for sure.